flocker/control/_config.py

```diff
@@ -210,8 +210,18 @@ def convert(self):
         if volume:
             config['volume'] = volume
         config['restart_policy'] = self.convert_restart_policy()
+        cpu_shares = self.convert_cpu_shares()
```

Comment:
I assume `ApplicationMarshaller` is used *somewhere* in the flocker-deploy execution path (`flocker-reportstate`, at a gues). So I'd rather not mess with it without adding tests for that code path... but that's a code path we want to kill.